corehq/apps/domain/forms.py

```diff
@@ -186,15 +186,13 @@ def __init__(self, *args, **kw):
             'description',
             'project_type',
             'image',
+            'documentation_file',
             'video',
             'share_multimedia',
             'share_reminders',
             'license',
             'cda_confirmed',]
 
-        if self.dom and toggles.DOCUMENTATION_FILE.enabled(user.username):
-            self.fields.keyOrder.insert(5, 'documentation_file')
```

Comment:
lol yup, figured though it was the easiest thing to remove when we removed the toggle